### PR TITLE
support show pump/drainer status && change pump/drainer's status

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 
support show pump/drainer status && change pump/drainer's status

### What is changed and how it works?
git cherry pick 
https://github.com/pingcap/parser/pull/217
https://github.com/pingcap/parser/pull/243

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by kennytm** [comment]
LGTM

File: parser.y [hunk 1]
```diff
@@ -6056,6 +6079,18 @@ ShowTargetFilterable:
 			Tp: ast.ShowProcedureStatus,
 		}
 	}
+|	"PUMP" "STATUS"
+	{
+		$$ = &ast.ShowStmt {
+			Tp: ast.ShowPumpStatus,
+		}
+	}
+|	"DRAINER" "STATUS"
```
Comment (zimulala) [inline parser.y]: Do we add a test to parser_test.go ?

Comment (WangXiangUSTC) [reply parser.y]: ok，I will add test in master, and then cherry-pick to release branch

Comment (zimulala) [reply parser.y]: No need to push a PR only for this, I just give a suggestion

**Pull request merged**